### ISSUE TITLE
Can't use jsx inside jsx files

### ISSUE THREAD

**Issue by asaadmahmood** [yannickcr/eslint-plugin-react#2129]
Can anyone tell me why I get this error:
"JSX not allowed in files with extension '.jsx'",

![screen shot 2019-01-10 at 7 31 56 am](https://user-images.githubusercontent.com/11034289/50942291-f55c3480-14a9-11e9-82c0-091604a65aa8.png)

**Comment by HerpGuy88** [comment]
I don't think this should have been closed... I'm having the same problem with a .jsx file. If you look at the error message, it shows the extension is .jsx

**Comment by ljharb** [comment]
Ah, true enough, @HerpGuy88 @asaadmahmood what are your eslint configs? Note that it can also be configured to force you to name your jsx files “.js”, and i believe CRA may do that.
